### DISCUSSION TITLE
Custom Save Path?

### DISCUSSION THREAD

**Discussion by y0himba** [hlky/stable-diffusion#293] [Q&A] [answered]
Hello. I'd like to specify a save path other than the default. What file(s) do I need to edit to change it?

**Comment by edgartaor** [comment, marked as answer]
In `/scripts/relaucher.py`

Change this to your custom path
`additional_arguments = "--outdir_txt2img=C:\\outputs\\txt2img --outdir_img2img=C:\\outputs\\img2img"`

**Reply by y0himba** [reply]
Thank you!
